MAINTAINERS.md

```diff
@@ -1,30 +1,18 @@
 # Maintainers
 
 See [Governance](Documentation/community/governance/commit_access.rst) for
-governance, commit, and vote guidelines as well as maintainer responsibilities.
+governance, commit, and vote guidelines as well as committer responsibilities.
 Everybody listed is a committer as per governance definition.
 
-All committers and maintainers have identical voting rights. Maintainers have
-additional administrative and janitorial responsibilities.
-
-Regardless of maintainer or committer status, any code contribution is subject
+Regardless of committer status, any code contribution is subject
 to code review requirements as per [CODEOWNERS](CODEOWNERS) and PRs get merged
 by the "janitor role" which is rotated between all committers.
```

Comment:
Nyc, but since you are updating the file, you might as well update this.

```suggestion
by the "tophat role" which is rotated between all committers.
```